notebooks/2-diverse-filetypes.ipynb

```diff
@@ -0,0 +1,336 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2072e221-3f2f-42ac-8d1c-81c6f93d3137",
+   "metadata": {},
+   "source": [
+    "_a small incantation so the paths make sense_"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "61bc5855-c4df-4b5f-b211-ab5cf9a75aad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.chdir(\"..\")"
```

Comment:
I briefly looked for a way to avoid this and yet keep the `data/[...]` paths but couldn't find anything easy